src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/endpoints/Endpoint.kt

```diff
@@ -39,8 +37,6 @@ data class Endpoint<TRoute>(
 
     override fun transform(x: Any) = when (x)
     {
-        is SplitData<*, *> -> x.serialize(Serializer.instance)
-        is DataTable<*> -> x.serialize(Serializer.instance)
         is AuthenticationResponse -> Serializer.instance.gson.toJson(x)!!
         else -> Serializer.instance.toResult(x)
```

Comment:
It would have been great if we could have done the streaming here, completely auto-magically. Unfortunately, the response transformer does not have access to the output stream.

Reply:
So what happens when Spark tries to transform a streamed response? Does it just never hit this method because `StreamedEndpoint.strream()` doesn't return anything?

Reply:
I think it does transform the response (which is just `kotlin.Unit`) but because we have already started writing to the response stream this output is discarded.

Reply:
Ok, I guess no harm in letting it do that.